deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -396,6 +396,15 @@ private FileObject addFileToWork(WorkObject work, Resource childResc)
         // Retry if there are checksum failures
         for (int retryCnt = 1; retryCnt <= CHECKSUM_RETRIES; retryCnt++) {
             try {
+                if (retryCnt > 1) {
+                    try {
+                        Thread.sleep(retryCnt * 1000);
+                    } catch (InterruptedException e) {
+                        log.debug("Retry ingest pause for {} due to a checksum mismatch, {} retries remaining: {}",
```

Comment:
rather than eating the interrupt, could you rethrow it as a `JobInterruptedException`? Shouldn't need an extra log statement in that case.